content/en/news/releases/1.11.x/announcing-1.11/change-notes/index.md

```diff
@@ -0,0 +1,248 @@
+---
+title: Change Notes
+linktitle: 1.11.0
+subtitle: Minor Release
+description: Istio 1.11.0 release notes.
+publishdate: 2020-08-10
+release: 1.11.0
+weight: 10
+aliases:
+    - /news/announcing-1.11.0
+---
+
+{{< warning >}}
+This is an automatically generated rough draft of the release notes and has not yet been reviewed.
+{{< /warning >}}
+
+# Release Notes
+
+## Deprecation Notices
+
+These notices describe functionality that will be removed in a future release according to [Istio's deprecation policy](/about/feature-stages/#feature-phase-definitions). Please consider upgrading your environment to remove the deprecated functionality.
+
+
+
+## Traffic Management
+
+
+- **Promoted** [CNI](/docs/setup/additional-setup/cni/) to beta ([Issue #86](https://github.com/istio/enhancements/issues/86)).
+
+- **Improved** resolution of headless services via in-agent DNS to include endpoints
+from other clusters that are on the same network.
+  ([Issue #27342](https://github.com/istio/istio/issues/27342))
+
+- **Improved** usage of AUTO_PASSTHROUGH Gateways to no longer require configuring the `ISTIO_META_ROUTER_MODE` environment variable on the gateway deployment; instead, it is automatically detected
+  ([Issue #33127](https://github.com/istio/istio/issues/33127))
+
+- **Improved** the XDS configuration generation to cache Envoy clusters, substantially improving performance of the control plane in many cases
+  ([Issue #26732](https://github.com/istio/istio/issues/26732))
+
+- **Improved** CNI network plugin to send logs to the CNI daemonset. This allows viewing CNI logs using `kubectl logs`, instead of looking at kubelet logs.
+  ([Issue #32437](https://github.com/istio/istio/issues/32437))
+
+- **Improved** service conflict resolution to favor Kubernetes Services over `ServiceEntries` with the same hostname.
+
+- **Updated** CNI install container and race condition repair container are combined into one container.
+  ([Issue #33712](https://github.com/istio/istio/issues/33712))
+
+- **Updated** the Istiod debug interface to be only accessible over localhost or with proper authentication (mTLS or JWT).
+The recommended way to access the debug interface is through `istioctl experimental internal-debug`, which handles
+this automatically.
+  
+
+- **Added** the `ShutdownDuration` flag to [pilot-discovery](/docs/reference/commands/pilot-discovery/) so that users can configure the duration istiod needs to terminate gracefully. The default value is 10s.
```

Comment:
```suggestion
- **Added** the `shutdownDuration` flag to [pilot-discovery](/docs/reference/commands/pilot-discovery/) so that users can configure the duration istiod needs to terminate gracefully. The default value is 10s.
```